### ISSUE TITLE
Add grain to _parse_basic_info()

### ISSUE THREAD

**Issue by cmuell89** [FraBle/python-duckling#7]
Grain is quite important for many datetime use cases. Could this be added to the DucklingWrapper method _parse_basic_info() if the results return a grain key-value pair.  The abstract DucklingWrapper is nice to use for convenience but limiting the grain makes its somewhat useless, for my use case at least.

I could make pull request to do so if you prefer.

**Comment by FraBle** [comment]
Pull requests are always welcome :)

**Comment by cmuell89** [comment]
Pulled down. Running tests.

**Comment by FraBle** [comment]
I integrated grain now into the wrapper as well. It will return something like
```json
    {
        "dim": "time",
        "end": 23,
        "start": 6,
        "text": "for 2 hours today",
        "value": {
            "grain": "minute",
            "others": [
                {
                    "grain": "minute",
                    "value": "2017-05-09T16:56:00.000-07:00"
                }
            ],
            "value": "2017-05-09T16:56:00.000-07:00"
        }
    }
```
Released with version [1.6.0](https://pypi.python.org/pypi/duckling/1.6.0)
